shared_model/cryptography/keypair.hpp

```diff
@@ -18,33 +18,39 @@
 #ifndef IROHA_SHARED_MODEL_KEYPAIR_HPP
 #define IROHA_SHARED_MODEL_KEYPAIR_HPP
 
+#include "cryptography/blob_impl.hpp"
 #include "cryptography/private_key.hpp"
 #include "cryptography/public_key.hpp"
-#include "interfaces/model_primitive.hpp"
+#include "interfaces/primitive.hpp"
 #include "utils/string_builder.hpp"
 
+#include "common/types.hpp"
+
 namespace shared_model {
   namespace crypto {
     /**
      * Class for holding a keypair: public key and private key
      */
-    class Keypair : public interface::ModelPrimitive<Keypair> {
+    class Keypair : public interface::Primitive<Keypair, iroha::keypair_t> {
```

Comment:
What is the reason for this change?

Reply:
Well, I changed it because we actually have an old "model" object for ```keypair``` - ```keypair_t``` is not a primitive type like ```blob<N>``` but a struct. So I decided to implement ```Primitive``` interface and ```makeOldModel``` to work with ```keypair``` more conveniently.